src/command.ts

```diff
@@ -1,26 +1,5 @@
-interface BaseCommand {
-  readonly name: string
-  readonly description: string
-  readonly aliases?: string[]
-}
-
-/**
- * Groups several LeafCommands
- */
-export interface GroupCommand extends BaseCommand {
-  subCommandClasses: { new (): Command }[]
-}
-
-/**
- * CLI Command Classes, which actually do something important
- */
-export interface LeafCommand extends BaseCommand {
-  run(): void | Promise<void>
-}
+import { GroupCommand, LeafCommand } from 'madlad'
 
-/**
- * Defines the Command classes can implement either Subcommand or GroupCommand
- */
 export type Command = LeafCommand | GroupCommand
```

Comment:
Maybe would it be better to use it across the code?
```suggestion
export type Command = (LeafCommand & Partial<{ [key: string]: any }>) | (GroupCommand & Partial<{ [key: string]: any }>)
```